WooCommerce/Classes/ViewRelated/Orders/Collect Payments/CollectOrderPaymentUseCase.swift

```diff
@@ -94,23 +98,23 @@ final class CollectOrderPaymentUseCase: NSObject, CollectOrderPaymentProtocol {
          stores: StoresManager = ServiceLocator.stores,
          paymentOrchestrator: PaymentCaptureOrchestrating = PaymentCaptureOrchestrator(),
          orderDurationRecorder: OrderDurationRecorderProtocol = OrderDurationRecorder.shared,
-         alertsPresenter: CardPresentPaymentAlertsPresenting? = nil,
-         preflightController: CardPresentPaymentPreflightControllerProtocol? = nil,
+         alertsPresenter: CardPresentPaymentAlertsPresenting,
+         tapToPayAlertsProvider: CardReaderTransactionAlertsProviding,
+         bluetoothAlertsProvider: CardReaderTransactionAlertsProviding,
+         preflightController: CardPresentPaymentPreflightControllerProtocol,
          analyticsTracker: CollectOrderPaymentAnalyticsTracking? = nil) {
         self.siteID = siteID
         self.order = order
         self.formattedAmount = formattedAmount
         self.rootViewController = rootViewController
         self.onboardingPresenter = onboardingPresenter
```

Comment:
nit: now that the preflight controller is being DI'ed, it looks like `onboardingPresenter` isn't needed anymore and we can consider removing it.